res/linuxPackaging.json

```diff
@@ -18,8 +18,8 @@
     "x-scheme-handler/ftp"
   ],
   "requires": [
-    "GConf2",
-    "libXScrnSaver",
+    "(GConf2 or gconf2)",
+    "(libXScrnSaver or libXss1)",
```

Comment:
I'll close this PR if someone opens PR with better solution.

For now let's put it on hold.